delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutes.scala

```diff
@@ -187,8 +187,10 @@ class AclsRoutes(identities: Identities, acls: Acls)(implicit
                     acls
                       .list(addressFilter)
                       .map { aclCol =>
-                        val filtered = aclCol.filterByPermission(caller.identities, aclsPermissions.read)
-                        SearchResults(filtered.value.size.toLong, filtered.value.values.toSeq)
+                        val accessibleAcls = aclCol.filterByPermission(caller.identities, aclsPermissions.read)
```

Comment:
We could add a test for this one so that it does not bite again

Reply:
All those methods are already tested. The `++`, the `filterByPermission`, the `filter`.
We could add a regression test in the integration tests, but I won't do it in this PR